src/test/java/com/redislabs/redistimeseries/RedisTimeSeriesTest.java

```diff
@@ -246,6 +246,21 @@ public void testAdd() {
       Assert.assertEquals(600L, client.add("seriesAdd", 600L, 1.1, 10000, null));
     }
 
+    Assert.assertEquals(
+        400L,
+        client.add(
+            "seriesAdd4",
+            400L,
+            0.4,
+            7898L /*retentionTime*/,
+            true,
+            1000L,
+            DuplicatePolicy.MAX,
+            labels));
```

Comment:
@gkorland @filipecosta90 Would you please re-evaluate google formatter? This is making the code way too long.

Reply:
@sazzad16 any formatter will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.

Reply:
Let's discuss this on a specific PR. Any format will do ( as long we follow it and enforce it on CI ). Please submit an alternative that you find most suited.